### ISSUE TITLE
Add Authentication layer to GraphDB

### ISSUE THREAD

**Issue by DinisCruz** [TeamMentor/TM_4_0_Design#290]
At the moment TM_Graph has no authentication and should only be accessible via the localhost (i.e. not exposed to the interwebs)

----
[TM-4.0-Security](https://github.com/TeamMentor/TM_4_0_Design/wiki/TM-4.0-Security)

**Comment by roman87** [comment]
Yep definitely important.  Is now part of a story. Removing priority.
